pom.xml

```diff
@@ -88,7 +88,7 @@
 
         <module.skipCopyDependencies>false</module.skipCopyDependencies>
 
-        <dev.images.postgres>postgres:16</dev.images.postgres>
+        <dev.images.postgres>quay.io/postgres/postgres:16</dev.images.postgres>
```

Comment:
is the potsgres image published on quay.io? we should use the "official" images wherever they are published

Reply:
@johnaohara TBH I don't have experience with quay.io, look at the CI seem it is pulling the images-https://github.com/Hyperfoil/Horreum/actions/runs/7237345338/job/19716890631#step:6:633
if there is no image then how it is able to pull?

Reply:
That build pulled  the `postgres:16`, which will have been resolved to docker.io.  

I took a look, the postrges community do not push to quay.io, so this image should stay as it is